modules/cluster-logging-collector-log-forwarding-supported-plugins.adoc

```diff
@@ -15,20 +14,36 @@ Red Hat tests each of the combinations shown in the following table. However, yo
 |====
 | Output types   | Protocols          | Tested with
 
-| fluentdForward | fluent forward v1  | fluentd 1.7.4
+| fluentdForward
+| fluentd forward v1
+a| fluentd 1.7.4
 
-| elasticsearch  | elasticsearch v?	  | Elasticsearch 6.8.1
+logstash 7.10.1
 
-| syslog         | RFC-3164, RFC-5424 | rsyslog 8.37.0-9.el7
+| elasticsearch
+| elasticsearch
+a| Elasticsearch 6.8.1
 
-| kafka          | kafka 0.11         | kafka 2.4.1
+Elasticsearch 7.10.1
+
+| syslog
+| RFC-3164, RFC-5424
+| rsyslog 8.37.0-9.el7
+
+| kafka
+| kafka 0.11
+| kafka 2.4.1
 
 |====
 
-// Note to tech writer, validate these items against the corresponding line of the test configuration file that Red Hat OpenShift Logging version 5.0 uses: https://github.com/openshift/origin-aggregated-logging/blob/release-5.0/fluentd/Gemfile.lock. This file is the authoritative source of information about which items and versions Red Hat tests and supports.
+// Note to tech writer, validate these items against the corresponding line of the test configuration file that Red Hat OpenShift Logging version 5.0 uses: https://github.com/openshift/origin-aggregated-logging/blob/release-5.0/fluentd/Gemfile.lock
+// This file is the authoritative source of information about which items and versions Red Hat tests and supports.
 // According to this link:https://github.com/zendesk/ruby-kafka#compatibility[Zendesk compatibility list for ruby-kafka], the fluent-plugin-kafka plug-in supports Kafka version 0.11.
 
-[NOTE]
-====
-Previously, the syslog output supported only RFC-3164. The current syslog output adds support for RFC-5424.
-====
+Comments:
+
+* Previously, the syslog output supported only RFC-3164. The current syslog output adds support for RFC-5424.
```

Comment:
This `Comments:` section isn't done anywhere else in our docs. Was there a specific reason why the admonition styling was removed? This seems like good text to call out in an admonition.

I would say if an admonition is not appropriate, to just add this info as regular text.

Reply:
Good point. I changed from admonition to comments because, at one point, there were going to be multiple comments, which aren't allowed inside a single admonition block, and we should avoid stacking multiple admonitions on top of eachother. Now, with only one comment, I'll revert this to being admonition block again.